tests/jsrun.py

```diff
@@ -86,10 +86,12 @@ def require_engine(engine):
     sys.exit(1)
 
 
-# TODO(sbc): Move to into test directory
-def run_js(filename, engine, args=[], check_timeout=False,
+def run_js(filename, engine=None, args=[], check_timeout=False,
            stdin=None, stdout=PIPE, stderr=None, cwd=None,
            full_output=False, assert_returncode=0, skip_check=False):
+  if not engine:
+    engine = shared.JS_ENGINES[0]
```

Comment:
this should probably be NODE_JS, but I agree we should not change that in this PR to keep it simple.